### PR TITLE
add img and gps icon

### PR DESCRIPTION
**Pull request by stef74**

### CODE REVIEW

**Comment by stef74** [comment]
some improvement needed

**Comment by stef74** [comment]
capture d ecran 2017-12-10 a 13 16 38
capture d ecran 2017-12-10 a 13 16 11

**Comment by stef74** [comment]
Is probaly enough for the 1st version ...

File: c2corg_ui/static/partials/cards/outings.html [hunk 1]
```diff
@@ -132,7 +132,13 @@
             </div>
           </a>
         </div>
-      </div>    
+      </div>
+      <div class="parent-bubble" ng-if="doc['img_count']">
+          <span class="glyphicon glyphicon-picture" uib-tooltip="({{::doc['img_count']}})" tooltip-append-to-body="true"></span>
```
Comment (brunobesson) [inline c2corg_ui/static/partials/cards/outings.html]: Wouldn't a text like `3 pictures for this outing` be better than `(3)`?

File: c2corg_ui/static/partials/cards/outings.html [hunk 2]
```diff
@@ -132,7 +132,13 @@
             </div>
           </a>
         </div>
-      </div>    
+      </div>
```
Comment (brunobesson) [inline c2corg_ui/static/partials/cards/outings.html]: this is not related to your PR, but there is a typo on line 80 (double quotes). Could you also correct this?

Comment (brunobesson) [reply c2corg_ui/static/partials/cards/outings.html]: Also, on line 128, the tag is not closed, the greater char is missing.

File: c2corg_ui/static/partials/cards/outings.html (hunk 1 above)
Comment (brunobesson) [inline c2corg_ui/static/partials/cards/outings.html]: indentation should be 2 spaces, not 4

Comment (stef74) [reply c2corg_ui/static/partials/cards/outings.html]: 3 picture(s) is probaly enough !

File: c2corg_ui/static/partials/cards/outings.html (hunk 2 above)
Comment (stef74) [reply c2corg_ui/static/partials/cards/outings.html]: fixed

File: c2corg_ui/static/partials/cards/outings.html (hunk 1 above)
Comment (stef74) [reply c2corg_ui/static/partials/cards/outings.html]: fixed

Comment (brunobesson) [reply c2corg_ui/static/partials/cards/outings.html]: Yeah, sure. If we could handle plurals by the same occasion :pray:

**Pull request merged**